orchagent/portsorch.cpp

```diff
@@ -1155,6 +1208,12 @@ bool PortsOrch::addBridgePort(Port &port)
 {
     SWSS_LOG_ENTER();
 
+    if (port.m_bridge_port_id)
+    {
+        SWSS_LOG_WARN("port %s already in 1Q bridge", port.m_alias.c_str());
```

Comment:
can be DEBUG LEVEL.

Reply:
there is back and forth for addBridgePort() and removeBridgePort().
Hope the following implementation is ok:
"Calling addBridgePort() only when bridge port not created for port/lag, and calling removeBridgePort() only when port/lag not in any VLAN"